CMakeLists.txt

```diff
@@ -16,8 +16,19 @@ find_package(YARP REQUIRED)
 list(APPEND CMAKE_MODULE_PATH ${YARP_MODULE_PATH})
 include(YarpPlugin)
 include(YarpInstallationHelpers)
+
 set(YARP_FORCE_DYNAMIC_PLUGINS TRUE)
 
+# Libraries type
+if(MSVC)
+ option(BUILD_SHARED_LIBS "Build libraries as shared as opposed to static" OFF)
+else()
+ option(BUILD_SHARED_LIBS "Build libraries as shared as opposed to static" ON)
+endif()
```

Comment:
I am not sure what is the effect of this. By default, `BUILD_SHARED_LIBS` is already set to `OFF`, and the plugin should be always built as dynamic libraries thanks to the earlier line:
~~~
set(YARP_FORCE_DYNAMIC_PLUGINS TRUE)
~~~